src/models/ActivitySchema.ts

```diff
@@ -1,8 +1,8 @@
 import * as Joi from 'joi';
 import { activitiesTypes, stationTypes, waitReasons } from './Activity';
-import { ActivityType } from '../assets/enums';
+import { ActivityType } from '@dvsa/cvs-type-definitions/types/v1/enums/activityType.enum';
 
-export const ActivitySchema = Joi.object().keys({
+export const ActivityJoiSchema = Joi.object().keys({
```

Comment:
Not a fan of this name, maybe activitiesSchema? visitSchema?

Reply:
Not sure. I feel activitiesSchema isn't distinct enough from ActivitySchema though. And visit is something else/more specific, no?